l10n_it_ricevute_bancarie/wizard/wizard_riba_issue.py

```diff
@@ -73,8 +73,8 @@ def create_rdl(countme, bank_id, rd_id, date_maturity, partner_id,
         countme = 1
 
         for move_line in move_lines:
-            if move_line.partner_id.bank_ids:
-                bank_id = move_line.partner_id.bank_ids[0]
+            if move_line.invoice_id.riba_partner_bank_id:
+                bank_id = move_line.invoice_id.riba_partner_bank_id
             else:
                 raise exceptions.Warning(
                     _('No bank has been specified for partner %s!') %
```

Comment:
Sarebbe bene anche adeguare il messaggio: "No bank has been specified for invoice %s"